arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -730,7 +738,11 @@
           "enum": [
             "key1",
             "key2"
-          ]
+          ],
+          "x-ms-enum": {
+            "name": "AccessKeyName",
+            "modelAsString": true
```

Comment:
I think you wont have more than 2 keys at any point in time. Primary and secondary. So having the "modelAsString": false over here would make sense. What do you think?